src/lambda_function.py

```diff
@@ -21,17 +22,46 @@ def lambda_handler(event, context):
     if (not validators.url(url)):
         return json.dumps({"error" : "Invalid URL"})
     
-    credibilityresult = cr.getCredibilityScore(url)
-    
+    #### Define the object skeleton
     object = {
         "url" : url,
-        "results" : [
-            credibilityresult
-        ]
+        "article" : {},
```

Comment:
I think we should actually remove this variable initialisation - it's created as part of the good or error case down below.